spec/steps/provision.fmf

```diff
@@ -68,3 +68,20 @@ example: |
             how: beaker
             family: Fedora31
             tag: released
+
+/connect:
+    summary: Connect to a provisioned box
+    description:
+        Do not provision a new system. Instead, use provided
+        authentication data to connect to a running machine.
+    example: |
+        provision:
+            how: connect
+            guest: hostname or ip address
+            user: username
```

Comment:
nor sure where to mention that default value is ```root```

Reply:
Included `root` default in the detailed description.

Reply:
thanks